build/docker/zap-baseline.py

```diff
@@ -422,6 +429,11 @@ def main(argv):
       logging.debug ('Ajax Spider complete')
 
     # Wait for passive scanning to complete
+    if (delay):
+      start_scan = datetime.now()
+      while ( (datetime.now() - start_scan).seconds < delay ):
+        time.sleep(5)
+        logging.debug ('Waiting for passive scan to finish: ' + str(delay - (datetime.now() - start_scan).seconds) + ' seconds left')
```

Comment:
This might be misleading since the the passive scanner might not have finished. I'd suggest using other phrase to indicate that's delaying the passive scan check.